api/models/user_custom_fields.py

```diff
@@ -8,14 +8,38 @@ class UserCustomFieldsQuery(BaseQuery):
     def get_by_user_phone(self, phone):
         return self.filter(UserCustomFields.user_phone == phone).all()
 
+    def mark_user_groups_as_inactive(self, phone):
+        self.filter(and_(UserCustomFields.user_phone == phone)).update(
+            {UserCustomFields.status: UserCustomFields.UserCustomFieldStatus.INACTIVE}
+        )
+
+    def set_custom_field_as_active(self, field_name, field_value, phone):
```

Comment:
Why we are using different terminology for activating and deactivating scenarios? Check the inactive method naming convention above.

Reply:
Corrected it.